config/jobs/kubernetes/kops/build_grid.py

```diff
@@ -197,7 +197,7 @@ def build_test(cloud='aws',
                test_parallelism=25,
                test_timeout_minutes=60,
                skip_override=None,
-               runs_per_day=None):
+               runs_per_day=-1):
```

Comment:
Would `0` also do the same thing? The condition I found seems to check for `> 0`.
https://github.com/kubernetes/test-infra/blob/f1f6f30cee229bfc87fb3173e201b7e2b08f60f2/config/jobs/kubernetes/kops/build_grid.py#L103